rllib/policy/torch_policy_v2.py

```diff
@@ -453,7 +457,7 @@ def optimizer(
             ]
         else:
             optimizers = [torch.optim.Adam(self.model.parameters())]
-        if getattr(self, "exploration", None):
+        if self.exploration:
```

Comment:
This might break for algos not using the standard `Algorithm` path, like ES, ARS, .... But tests seem to be ok.